### ISSUE TITLE
General Utils

### ISSUE THREAD

**Issue by MertCokelek** [MertCokelek/VisualSaliencyTransformer#4]
**General""
- nn.Linear
- nn.Dropout
- nn.LayerNorm
- nn.GeLU
- nn.Sigmoid
- nn.Fold
- nn.Unfold
- nn.init.orthogonal
- torch.einsum

**transformer_block**
- softmax
- sinusoidal encoding

**Comment by MertCokelek** [comment]
Completed:
* Linear
* Dropout
* LayerNorm
* GeLU
* Einsum
* Softmax
* Sinusoidal
**TODO**
* Sigmoid
* Orthogonal init
* Softmax
